service/Dockerfile

```diff
@@ -2,10 +2,12 @@
 FROM openjdk:8-jre
 
 WORKDIR /usr/local/src
-COPY . /usr/local/src
+COPY . /usr/local/src/
```

Comment:
The `.dockerignore` removes all of the unnecessary files from the build context; The files below are the only ones present for the global copy:

```
.
├── Dockerfile
├── graph
│   ├── transit.edges
│   ├── transit.graph
│   └── transit.vertices
├── production.json
├── setup.py
├── src
│   └── main
│       └── resources
│           ├── application.conf
│           └── webapp
└── target
    └── scala-2.10
        ├── classes
        │   ├── application.conf
        │   ├── geotrellis
        │   └── webapp
        └── geotrellis-transit-assembly-0.1.0-SNAPSHOT.jar
```
I have the `COPY` and `RUN` lines below because everything besides `./target` belongs at `/usr/local/src` and I thought  `COPY`ing each file into the container would take up too many layers.